src/WLR/Types/Buffer.hsc

```diff
@@ -26,3 +28,13 @@ import WLR.Util.Addon
     events release, WL_signal,
     addons, WLR_addon_set
 }}
+
+{{ struct
+    include/wlr/types/wlr_buffer.h,
```

Comment:
None of the other include paths start with `include/`; shouldn’t this just be `wlr/types/wlr_buffer.h`?